Vision/tests/Unit/ImageTest.php

```diff
@@ -157,6 +157,6 @@ public function testUrlSchemes()
         $this->assertEquals($urls[0], $images[0]->requestObject()['image']['source']['imageUri']);
         $this->assertEquals($urls[1], $images[1]->requestObject()['image']['source']['imageUri']);
         $this->assertEquals($urls[2], $images[2]->requestObject()['image']['source']['imageUri']);
-        $this->assertFalse(isset($images[3]->requestObject()['image']['source']['imageUri']));
+        $this->assertArrayNotHasKey('imageUri', $images[3]->requestObject()['image']['source']);
```

Comment:
`source` may also not exist, causing this test to fail. We should update this to:

```php
$this->assertArrayNotHasKey('source', $images[3]->requestObject()['image']);
```